packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.integration.test.tsx

```diff
@@ -25,8 +25,10 @@ import {
   getUseQueryLoadingMock,
   getUseQuerySuccessMock,
 } from '../../testutils/ReactQueryMockUtils'
+import { getEntityHandlers } from '../../mocks/msw/handlers/entityHandlers'
+import { MOCK_REPO_ORIGIN } from '../../utils/functions/getEndpoint'
 
-const onExportToCavatica = jest.fn()
+const onExportToCavatica = jest.fn().mockImplementation(() => Promise.resolve())
```

Comment:
Fix a bug I introduced in a previous PR.  On success, the export dialog is closed